### ISSUE TITLE
Every InlineSVG is missing an aria-label

### ISSUE THREAD

**Issue by ashu8912** [processing/p5.js-web-editor#1259]
- Found a bug
Right Now Every SVG used is missing an aria-label/tooltip when we hover over an SVG it says triangle, a wrapper over svgs, like button has an aria-label but hovering over the  SVG shows the aria-label on the SVG which is by default set to triangle.
This feature can be easily added by providing title as a prop on the InlineSVG component.
